### ISSUE TITLE
Error running Quick Encounter with Foundry 10 Testing 1 (v10.275)

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#91]
Open a Quick Encounter Journal Entry and press the [Run Quick Encounter] button and get:
```
commons.js:3462 Uncaught (in promise) TypeError: Cannot assign to read only property '_id' of object '#<TokenDocument5e>'
    at SchemaField._cleanType (commons.js:3462:20)
    at SchemaField.clean (commons.js:3194:19)
    at TokenDocument.cleanData (commons.js:4781:26)
    at TokenDocument._initializeSource (commons.js:4768:31)
    at new DataModel (commons.js:4655:21)
    at new Document (commons.js:5182:7)
    at new BaseToken (commons.js:9448:3)
    at new ClientDocumentMixin (foundry.js:12575:5)
    at new <anonymous> (foundry.js:11748:5)
    at new TokenDocument (foundry.js:21002:5)
```

**Comment by spetzel2020** [comment]
This results from token document creation calls in ` async generateFullExtractedActorTokenData() {` :

```
  const tempTokenData = await actor.getTokenData(tokenData);
                    tempToken = new TokenDocument(tempTokenData, {actor: actor});
```
However, tempTokenData._id === null in the console, so??

**Comment by spetzel2020** [comment]
This is a known bug in Foundry v10.275; I plan to wait on the next test build to see if it's resolved, rather than workaround it (presumably by `delete`-ing the `_id` property

**Comment by spetzel2020** [comment]
Associated with https://github.com/foundryvtt/foundryvtt/issues/7766 ; Fixed in QE 1.0.4 by not calling the (now extraneous) TokenDocument() call

**Comment by spetzel2020** [comment]
Fixed in 1.0.4b

**Comment by spetzel2020** [comment]
Released in 1.0.4
